src/Widgets/ColumnContainerWidget/ColumnsEditorTab.scss

```diff
@@ -575,6 +575,13 @@
   color: #fff;
 }
 
+// -- generic scrivito_disabled editors -- //
+
+.scrivito_detail_content .scrivito_disabled {
+  opacity: 0.7;
+  cursor: not-allowed;
+}
+
```

Comment:
* [ ] Generic (shared) extension styles should be in the shared `src/assets/stylesheets/scrivitoExtensions.scss`.